<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>393</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>353</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Linear Probing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Displaced Probing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Secondry Clustering</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Double Hashing</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Store</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>K for sc:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox">
              <property name="value">
               <number>13</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Choose file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>K for dp:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_2">
              <property name="value">
               <number>7</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>member id</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>surname</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
         </layout>
        </item>
        <item row="0" column="4">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>country</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
         </layout>
        </item>
        <item row="0" column="5">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>email</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
         </layout>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>total items</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_9"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>average probing</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>total probing    </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
